bucket4j/src/test/java/com/linecorp/armeria/server/throttling/tokenbucket/BandwidthLimitTest.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2020 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.throttling.tokenbucket;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.Test;
+
+import io.github.bucket4j.Bandwidth;
+
+public class BandwidthLimitTest {
+
+    @Test
+    public void testConstructor() {
+        final BandwidthLimit bl1 = BandwidthLimit.of(100L, 1000L, 50L, Duration.ofSeconds(60L));
+        System.out.println(bl1);
+        assertEquals(100L, bl1.limit());
+        assertEquals(1000L, bl1.overdraftLimit());
+        assertEquals(50L, bl1.initialSize());
+        assertEquals(Duration.ofSeconds(60L), bl1.period());
```

Comment:
Could we use Assert4J instead of JUnit assertions? e.g.

    assertThat(bl1.limit()).isEqualTo(100L);

Reply:
Sorry. This was automatic. Not very used to AssertJ. There is nothing in particular in your guidelines about it. Maybe you should fix the guidelines if it's absolutely required.
Fixed.

Reply:
Thanks a lot for your patience and suggestion. Let me update our `CONTRIBUTING.md`. :bowing_man:

Reply:
https://github.com/line/armeria/pull/2569